cmd/crowdsec-cli/utils.go

```diff
@@ -112,12 +112,8 @@ func compAllItems(itemType string, args []string, toComplete string) ([]string,
 	comp := make([]string, 0)
 	hubItems := cwhub.GetHubStatusForItemType(itemType, "", true)
 	for _, item := range hubItems {
-		if toComplete == "" {
```

Comment:
strings.Contains(item.Name, toComplete) is always true if toComplete is empty string, so we can eliminate this condition branch